src/tz390.java

```diff
@@ -1710,7 +1711,7 @@ public  class  tz390 {
      String[]   op_table_370_notsupported = // Table added for RPI 1209A
         {"RIO      S    9C02 D2(B2)",
          "CLRCH    S    9F01 D2(B2)",
-         "CONCS    S    B200 D2(B2)",
+         // DSH #414 new z16 LBEAR replaces old nonsupported "CONCS    S    B200 D2(B2)",
```

Comment:
LBEAR is defined in the table for z16 instructions. Please leave CONCS as it was. We want to keep track f unsupported instructions and their position in the sequence of PoP manuals. That should obviate the need to go find all the details in case we need to implement the CONCS instruction...